docs/react/README.md

```diff
@@ -90,9 +204,39 @@ Included components:
 
 Learn more about the available components [here](./components.md).
 
+```tsx
+import {HatchifyProvider} from "@hatchifyjs/react"
+
+// Define Schemas
+const schemas = { ...Todo }
+
+// Create the Hatched React App instance
+const hatchedReact = hatchifyReact(createJsonapiClient("/api", Schemas))
+
+// Define variables for your Hatchify schema generated components
+const TodoDataGrid = hatchedReact.components.Todo.DataGrid
+
+const TodoColumn = hatchedReact.components.Todo.Column
+
+const TodoEmpty = hatchedReact.components.Todo.Empty
+
+// Render your custom component 
+const HatchedComponent = () => {
+  return (
+    <HatchifyProvider>
+      <TodoEmpty />
+      <TodoDataGrid>
+        <TodoColumn />
+      </TodoDataGrid>
+      <TodoEmpty />
```

Comment:
`TodoEmpty` is a compound component used for customizing a `TodoDataGrid` - it's what you want to render if the list is empty (no records found. `TodoColumn` is used to customize an existing column or add one, so i'd do something like:

```
<TodoDataGrid>
  <TodoEmpty>No todos found!</TodoEmpty>
  <TodoColumn
    label="Actions"
    renderDataValue={() => <button onClick={({ record }) => console.log(record)}>Log Todo</button>
  />
</TodoDataGrid>
```